traja/models/train.py

```diff
@@ -159,12 +161,16 @@ def fit(
 
         train_loader = dataloaders["train_loader"]
         test_loader = dataloaders["test_loader"]
+        if 'validation_loader' in dataloaders:
+            validation_loader = dataloaders['validation_loader']
+        else:
+            validate_every = None
```

Comment:
Could you add a test for the "validation data not present" situation, please? That way, no one can remove the functionality without noticing in the future.

Reply:
What do we want to do if validation data is not present? We shouldn't throw an error sometimes it is ok to not have a validation set.

Reply:
Print nothing about validation, is what I'd expect.